app/models/api_user.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
 
+require "base64"
+require "zlib"
+
 class ApiUser < ApplicationRecord
+  include StoreModel::NestedAttributes
+
+  TOKEN_FORMAT = /\Abops_[a-zA-Z0-9]{36}[-_a-zA-Z0-9]{6}\z/
+
   attribute :file_downloader, FileDownloaders.to_type
+  alias_attribute :value, :token
```

Comment:
Ideally we'd rename this model as `ApiToken` as it's not really a user and to that aim I've aliased the `token` to `value` so we don't have `token.token` where it's used.